### PR TITLE
added sbom visualization code

### PR DESCRIPTION
**Pull request by pramit-d**
added code to assessment_report.js & version_summary.js
https://github.com/Be-Secure/BeSLighthouse/issues/433

### CODE REVIEW

File: bes_theme/assets/javascripts/assessment_reports.js [hunk 1]
```diff
@@ -143,8 +156,11 @@ function fetch_json()
   console.log("report:"+report);
   if (report == "codeql" || report == "sonarqube") { // The sast reports(codeql, sonarqube, ...) are under sast dir.
     url = 'https://raw.githubusercontent.com/Be-Secure/besecure-assessment-datastore/main/'+ ossp_name + '/' + version + '/sast' + '/' + ossp_name+ '-' + version + '-' + report + '-report.json';  
-  }
 
+  } 
+  else if (report == "sbom") { // sbom dir
```
Comment (asa1997) [inline bes_theme/assets/javascripts/assessment_reports.js:161]: @pramit-d This check is not required.

File: bes_theme/assets/javascripts/version_summary.js [hunk 2]
```diff
@@ -110,11 +110,15 @@ function check_url(id, version, name)
   {
 
     var url = "https://raw.githubusercontent.com/Be-Secure/besecure-assessment-datastore/main/"+name+"/"+version+"/sast/"+name+"-"+version+"-"+id+"-report.json"
-  } else if (id == "fossology")
+  }
+  else if(id == "sbom"){
```
Comment (asa1997) [inline bes_theme/assets/javascripts/version_summary.js:114]: @pramit-d This check is not required.

**Comment by pramit-d** [comment]
> @pramit-d Please address the comments.

updated

**Review by asa1997 — APPROVED** [review]
@pramit-d @harimohanr LGTM

**Comment by asa1997** [comment]
@harimohanr 
image

File: bes_theme/assets/javascripts/assessment_reports.js [hunk 3]
```diff
@@ -117,6 +117,19 @@ function print_sonarqube_report(data) {
   document.getElementById("reports").innerHTML = table;
 }
 
+
+//print sbom report
+function print_sbom_report(data) {
+  json_data = JSON.stringify(data);
+  json_object = JSON.parse(json_data);
+  var table = "<table id=table>"
+  table += "<tr><th>Package name</th><th>version</th><th>supplier</th><th>downloadLocation</th><th>license</th></tr>"
```
Comment (asa1997) [inline bes_theme/assets/javascripts/assessment_reports.js:126]: - version -> Version
- supplier -> Supplier
- downloadLocation -> Download Location
- license -> License

Comment (asa1997) [reply bes_theme/assets/javascripts/assessment_reports.js:126]: @pramit-d Please do make these changes as well.

**Review by asa1997 — APPROVED** [review]
@pramit-d LGTM

**Review by asa1997 — CHANGES_REQUESTED** [review]
@pramit-d Please resolve the conflicts

**Comment by pramit-d** [comment]
> @pramit-d Please resolve the conflicts

@asa1997 working on that

**Review by asa1997 — APPROVED** [review]
@pramit-d @harimohanr LGTM

**Pull request merged**